docs/layers.md

```diff
@@ -52,6 +52,18 @@ For features in the `boundaries` layer, there are two additional variants `name:
 * `name:left:*` _See planned bug fix [#1102](https://github.com/tilezen/vector-datasource/issues/1102)._
 * `name:right:*` _See planned bug fix [#1102](https://github.com/tilezen/vector-datasource/issues/1102)._
 
+##### Chinese name localization
+Both `name:zh` and `name:zht` properties are best-effort. It means `name:zh` can sometimes contain Traditional Chinese and `name:zht` can sometimes contain Simplified Chinese. And we don't auto-translate Traditional Chinese to Simplified Chinese or vice versa.
+
+###### OSM
+We use three fields `name:zh`, `name:zh-Hans` and `name:zh-Hant` from OSM to populate two properties:
```

Comment:
Nit: add extra line between the header and the paragraph text to match the Markdown formatting used in this doc.